packages/web-app-files/src/composables/actions/files/useFileActionsDownloadArchive.ts

```diff
@@ -68,6 +90,12 @@ export const useFileActionsDownloadArchive = ({ store }: { store?: Store<any> }
         label: () => {
           return $gettext('Download')
         },
+        disabledTooltip: ({ resources }) => {
+          return archiverLimitsExceeded(resources)
+            ? $gettext('Unable to download that many files at once')
```

Comment:
Open for suggestions on the final text here btw 😄

Reply:
Not sure if the average non-tech user knows how to parse `allowed archive size`, but let's roll with it since feedback can easily be implemented at a later point